tox.ini

```diff
@@ -27,6 +27,19 @@ commands =
     python setup.py sdist
     twine check dist/*
 
+[testenv:watch-docs]
+skip_install = True
+deps =
+    -rdocs/requirements.txt
+    sphinx-autobuild
+commands =
+    sphinx-autobuild -W -b html -d {envtmpdir}/doctrees \
+        {posargs:-H 127.0.0.1} \
+        # HACK: Detect changes in README.rst; results in superfluous builds
+        # Mitigated somewhat by ignoring dotfiles
+        -z . -i '.*' \
+        docs docs/_build/html
+
```

Comment:
I've been wanting to do this for awhile, and it probably should have been its own PR, but it was really handy for this update, so I left it in.

Reply:
I'd rather discuss this in a separate pull request, myself. Specifically, what's the value of separating this out into a separate environment?

Reply:
I moved this to https://github.com/pypa/twine/pull/644